aspnet/web-api/overview/older-versions/using-web-api-1-with-entity-framework-5/using-web-api-with-entity-framework-part-3.md

```diff
@@ -1,8 +1,8 @@
 ---
 uid: web-api/overview/older-versions/using-web-api-1-with-entity-framework-5/using-web-api-with-entity-framework-part-3
 title: "Part 3: Creating an Admin Controller | Microsoft Docs"
-author: MikeWasson
-description: ""
+author: Rick-Anderson
+description: "In this section, we'll add a Web API controller that supports CRUD (create, read, update, and delete) operations on products. "
```

Comment:
suggestion: Learn how to add a Web API controller that supports CRUD (create, read, update, and delete) operations on products.